bower.json

```diff
@@ -1,30 +1,53 @@
 {
   "name": "lazy-imports",
+  "version": "2.0.0",
```

Comment:
Open Question to @rictic: I have this numbered at `2.0.0` because that was what all the other elements have been numbered for hybrid. However, there is no tags outside of `0.0.z` so should this be `1.0.0`?